### ISSUE TITLE
python-libxml2 v.2.9.4 not recognized on Debian sid

### ISSUE THREAD

**Issue by nbeaver** [FontManager/master#19]
When trying to build from source on Debian sid, `python-libxml2` is at version 2.9.4 and is installed under `/usr/lib/python2.7/dist-packages`. However, the `configure` step fails with this error message:

    checking for XML... no
    configure: error: Package requirements (libxml-2.0 >= 2.9) were not met:
    
    No package 'libxml-2.0' found

See attached for full log.

[config.log.txt](https://github.com/FontManager/master/files/417198/config.log.txt)

[policy.txt](https://github.com/FontManager/master/files/417197/policy.txt)

**Comment by JerryCasiano** [comment]
Python is not a requirement.

**Comment by JerryCasiano** [comment]
If that fails to build a Debian package for you, only been tested on Ubuntu.

You'll need to look at debian/control and manually install listed dependencies.

You should also ask your distribution update their packages.
The version in their repositories had been deprecated for years now.

**Comment by nbeaver** [comment]
Thanks, I was able to get it to build. I've attached the control file in case anyone else needs the list of dependencies.

The current Debian release (Jessie) is on  libgtk-3-0=3.14.5-1+deb8u1 but font-manager 0.7.3 requires gtk+-3.0 >= 3.16, so it probably won't be backported to the current release. However, the next release (Stretch) will be on libgtk-3-0=3.20.7 or higher, so hopefully the maintainer will update it by then.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=796817

Right now the old version is failing to build and it's less than a week away from being purged from the Debian repositories.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=830469

[control.txt](https://github.com/FontManager/master/files/418274/control.txt)
